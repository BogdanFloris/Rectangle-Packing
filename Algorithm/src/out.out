fixed
22
true
6
12 8
10 9
8 12
16 3
4 16
10 6
