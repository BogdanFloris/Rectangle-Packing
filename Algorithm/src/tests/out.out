container height: free
rotations allowed: no
number of rectangles: 6
12 8
10 9
8 12
16 3
4 16
10 6
placement of rectangles
0 0
0 8
12 0
0 18
20 0
10 12